src/components/main/HomeScreen.js

```diff
@@ -1,355 +1,288 @@
-                {/* Update isAuthenticated on navigation refresh */}
-                <NavigationEvents onWillFocus={() => { this.checkAuth(); this.selectBackground(); }} />
-                <View style={styles.buttonBarNav}>
-                  {/* <Text style={styles.bigGreenText}>&#x2013; &#x2013;</Text> //To be used after beta */}
-                  {/* Bug Reports to be used only for beta */}
-                  <TouchableOpacity activeOpacity={0.9}
-                    onPress={() => this.handleBugReports()}
-                    style={styles.navStyle}>
-                    <Ionicons style={styles.navigationIcon} name="ios-bug" />
-                  </TouchableOpacity>
-                  {/* isAuthenticated: false */}
-                  {!this.state.isAuthenticated &&
-                    <TouchableOpacity activeOpacity={0.9}
-                      onPress={() => this.props.navigation.navigate('SignIn')}
-                      style={styles.navStyle}>
-                      <FontAwesome style={styles.navigationIcon} name="user-circle-o" />
-                      <Text style={styles.navText}>
-                        {i18n.t('SIGN IN')}
-                      </Text>
-                    </TouchableOpacity>
-                  }
-                  {/* isAuthenticated: true */}
-                  {this.state.isAuthenticated &&
-                    <TouchableOpacity activeOpacity={0.9}
-                      onPress={() => this.props.navigation.navigate('UserProfile')}
-                      style={styles.navStyle}>
-                      <FontAwesome style={styles.navigationIcon} name="user-circle-o" />
-                      <Text style={styles.navText}>
-                        {i18n.t('PROFILE')}
-                      </Text>
-                    </TouchableOpacity>
-                  }
-                </View>
-                <View style={styles.appName}>
-                  <Text style={styles.bigWhiteText}>refloresta</Text>
+      <TouchableWithoutFeedback onPress={Keyboard.dismiss}>
+        <KeyboardAvoidingView
+          behavior={Platform.OS == "ios" ? "padding" : null}
+          enabled="false"
+          style={styles.container}>
+          <ImageBackground
+            source={this.background}
+            style={styles.imageBackground}>
+            <View>
+              <View style={styles.textContainer}>
+                {/* Intro Text Body View */}
+                <View>
+                  <Text style={styles.largeWhiteText} numberofLines={2}>
+                    A cada $6 uma árvore é plantada.
+                  </Text>
+
+                  <Text style={styles.mediumWhiteText}>
+                    Faça sua doação e ajude a recuperar a Mata Atlântica da
+                    Fazenda do Bulcão.
+                  </Text>
+
+                  <Text style={styles.smallWhiteText} numberOfLines={2}>
+                    Insira o número de vôo para iniciar ou
+                  </Text>
```

Comment:
2 lines?

Reply:
I decided to take out the line specifics, because it will wrap anyway with different translations